arviz/plots/backends/bokeh/khatplot.py

```diff
@@ -130,7 +137,7 @@ def plot_khat(
     xmax = len(khats)
 
     if show_bins:
-        bin_edges = np.array([ymin, 0.5, 0.7, 1, ymax])
+        bin_edges = np.array([ymin, threshold, 1, ymax])
```

Comment:
```suggestion
        bin_edges = np.array([ymin, good_k, 1, ymax])
```

The goal for this is to count how many values lie within each region visually separated with hlines, so it should match the horizontal lines being plotted. It should also be modified in the matplotlib version